app/components/TimezoneDropdown.js

```diff
@@ -25,6 +25,16 @@ var cx = require('classnames');
 var styles = require('../../styles/components/TimezoneDropdown.module.less');
 
 class TimezoneDropdown extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
Don't actually need to pass `props` to `super` here, but not hurting anything either.

Reply:
Actually, NVM. I see that they recommend it in the React docs.
`Class components should always call the base constructor with props`

Currently, it works without it, but should leave it as per their recommendations.